Plugins~/Src/MeshSyncClientBlender/MeshSyncClientBlender.cpp

```diff
@@ -170,7 +170,7 @@ PYBIND11_MODULE(MeshSyncClientBlender, m)
             BindMethod(Destroy, [](self_t& self) { self->Destroy(); })
             BindMethod(setup, [](self_t& self, py::object ctx) { bl::setup(ctx); })
             BindMethod(clear, [](self_t& self) { self->clear(); })
-            BindMethod(exportUpdatedObjects, [](self_t& self) { self->sendObjects(MeshSyncClient::ObjectScope::Updated, false); })
+            BindMethod(exportUpdatedObjects, [](self_t& self) { self->sendObjects(MeshSyncClient::ObjectScope::Updated, false); self->requestServerInitiatedMessage(); })
```

Comment:
We should probably separate this into a separate method, or change the name of the bound method.

Also, `requestServerInitiatedMessage` is old. Have you reflected your changes on the MeshSync side ?